### PR TITLE
Add Initial Content

### PR DESCRIPTION
**Pull request by blink1073**
@ellisonbg, @jasongrout, @sccolbert, am I on the right track with this?

### CODE REVIEW

**Comment by blink1073** [comment]
Specifically `kernel.ts`

File: package.json [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+    "plugin"
+  ],
+  "files": [
+    "lib/*.js",
+    "lib/*.d.ts"
+  ],
+  "phosphor-plugin": {
+    "extensionPoints": [
+      {
+        "point": "services:kernel",
```
Comment (sccolbert) [inline package.json]: shouldn't this be "id" instead of "point" ?

Comment (sccolbert) [inline package.json]: Do we want to standardize that namespace == package name?

File: scripts/tdoptions.json [hunk 2]
```diff
@@ -0,0 +1,16 @@
+{
+  "module": "commonjs",
+  "target": "ES5",
+  "excludeNotExported": true,
+  "experimentalDecorators": true,
```
Comment (sccolbert) [inline scripts/tdoptions.json]: no decorators

File: package.json (hunk 1 above)
Comment (blink1073) [inline package.json]: Not necessarily, it could be more granular that that (e.g. phosphide).

Comment (blink1073) [inline package.json]: Unless you had `phoshide:menu:items`, I suppose.

Comment (sccolbert) [inline package.json]: Yeah. I would use `phosphide:menu-bar`, `phosphide:context-menu`, etc...

**Comment by sccolbert** [comment]
lgtm for front matter.

File: src/tsconfig.json [hunk 3]
```diff
@@ -0,0 +1,21 @@
+{
+  "compilerOptions": {
+    "experimentalDecorators": true,
```
Comment (jasongrout) [inline src/tsconfig.json]: no decorators, like @sccolbert said.

**Comment by blink1073** [comment]
Okay, I'm done with the initial content.

**Comment by jasongrout** [comment]
Great, thanks!

**Pull request merged**